build.sbt

```diff
@@ -1,24 +1,24 @@
 import com.lucidchart.sbtcross.{Axis, CrossableProject, DefaultAxis}
-import com.typesafe.sbt.packager.archetypes.ServerLoader
+// import com.typesafe.sbt.packager.archetypes.ServerLoader
 import play.api.libs.json.Json
 
 val distributionAxis = new DefaultAxis {
   def name = "dist"
   def major(version: String) = version
 }
 
-lazy val admin = project.dependsOn(worker_2_11).settings(scalaVersion := "2.11.8")
+lazy val admin = project.dependsOn(worker_2_12).settings(scalaVersion := "2.12.11")
 // there's probably a better way to also produce a Systemd deb
-lazy val `admin-xenial` = admin.copy(id = "admin-xenial").settings(
-  serverLoading in Debian := ServerLoader.Systemd,
-  target := baseDirectory.value / "target-xenial",
-  scalaVersion := "2.11.8"
-)
-lazy val `admin-bionic` = admin.copy(id = "admin-bionic").settings(
-  serverLoading in Debian := ServerLoader.Systemd,
-  target := baseDirectory.value / "target-bionic",
-  scalaVersion := "2.11.8"
-)
+// lazy val `admin-xenial` = admin.copy(id = "admin-xenial").settings(
```

Comment:
I was meaning to dig into fixing these, after getting piezo-admin building successfully.

Reply:
we're going to do debs differently now. (I'll do that in a different PR).